### ISSUE TITLE
New report format 10 with Debian Buster (puppet 5.5.10)

### ISSUE THREAD

**Issue by larsen0815** [sodabrew/puppet-dashboard#394]
I recently upgraded my server to Debian Buster and noticed a warning in "/usr/share/puppet-dashboard/log/production.log":
````
W, [2019-10-22T14:43:24.262712 #32967]  WARN -- : Sanitizer: Unknown report format 10, using format 9 and hoping for the best
````
Seems to be no problem though, as the host is being reported as ok/unchanged.

**Comment by ZeroPointEnergy** [comment]
Yes, looks like only a new field was added so it just doesn't get recorded in the database but should not cause any problems otherwise.
